util/systemd/systemd_linux.go

```diff
@@ -18,6 +18,25 @@ var getNotifyOnce struct {
 	v *sdnotify.Notifier
 }
 
+type logOnce struct {
+	sync.Once
+}
+
+func (l *logOnce) log(format string, args ...interface{}) {
```

Comment:
`logf` instead of `log` (some Go tooling uses the combination of func-ending-in-`f` + signature `string+...interface{}` to do printf-style vet checks)